src/components/TodoTemplate.js

```diff
@@ -0,0 +1,42 @@
+/*투두 리스트의 레이아웃(흰색 박스)*/
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import TodoInput from './TodoInput';
+import TodoList from './TodoList';
+function TodoTemplate() {
+  //새로고침 해도 저장이 되게끔 localStorage에 저장
+  const [todoList, setTodoList] = useState(() =>
+    JSON.parse(window.localStorage.getItem('todo'))
+  );
+  useEffect(() => {
+    window.localStorage.setItem('todo', JSON.stringify(todoList));
+  }, [todoList]);
+
+  return (
+    <TodoTemplateBlock>
+      <TodoList
+        title={'대기중'}
+        todoList={todoList}
+        setTodoList={setTodoList}
+        isDone={false}
+      ></TodoList>
+      <TodoList
+        title={'완료됨'}
+        todoList={todoList}
+        setTodoList={setTodoList}
+        isDone={true}
+      ></TodoList>
```

Comment:
```suggestion
      <TodoList
        todoList={todoList}
        setTodoList={setTodoList}
      ></TodoList>
```

이런식으로 TodoList를 만들고 완료 여부에 따라 가르는 것은 컴포넌트 내부에서 처리했으면 컴포넌트를 좀 더 추상화했었을 수 있을것 같습니다. 개인적으로 코드가 중복된다는 느낌이 있네요

Reply:
자기 전에 피드백을 봐버려서 빠르게 빌드 에러를 수정하고 왔습니다🥺 시간 나실 때 테스트 한 번 해주세요ㅎㅎ 나머지 피드백은 내일 읽어보도록 하겠습니다..!

Reply:
라고 했지만... 피드백이 많지 않아 그냥 다 읽어버렸네요..! 이 부분도 내일 더 고민해보도록 하겠습니다! 피드백 감사합니다 🙇‍♀️👍